.github/pull_request_template.md

```diff
@@ -4,6 +4,6 @@
 
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
```

Comment:
Need to also mention checking against the current develop branch of connect (or modify testing workflow to run against the develop branch of connect, perhaps?)

Reply:
I think Lens should always be run against current develop version of Connect. A release QA should make sure Connect and Lens stable versions are working together

Reply:
We should update the requirements of Lens to reflect that. The developer of Lens should not _need_ to be testing against different versions of Connect. Just whatever version Lens requires.

Reply:
I'm concerned this is going to lead to review blocks between the two packages.
Say some bug in Connect causes a Lens test to fail (something with Container constructors, for example, since those are used by evaluators).
In that case the Lens PR might be "correct" per visual review and local testing with the Connect bugfix branch, but can't be merged (and probably shouldn't be approved) until the Connect bugfix is merged.

This seems like it's outside the scope of the present PR, but it's definitely something to keep on our radar.